src/radarChart.ts

```diff
@@ -1088,7 +1302,7 @@ module powerbi.extensibility.visual {
          * This function returns the values to be displayed in the property pane for each object.
          * Usually it is a bind pass of what the property pane gave you, but sometimes you may want to do
          * validation and return other values/defaults
-         * 
+         *
          * TODO: We should use SettingsParser instead. Please rewrite it in future versions.
          */
         public enumerateObjectInstances(options: EnumerateVisualObjectInstancesOptions): VisualObjectInstanceEnumeration {
```

Comment:
We should use `DataViewObjectParser` to simplify it. Let's include it to our backlog.